sklearn/decomposition/tests/test_kernel_pca.py

```diff
@@ -181,13 +181,13 @@ def test_kernel_pca_linear_kernel(solver, n_features):
     )
 
 
-def test_kernel_pca_n_components():
+def test_kernel_pca_n_components(global_random_seed):
```

Comment:
Seems `global_random_seed` is not needed for this test, since it only tests the output has the correct shape depending on `n_components`.

Reply:
Yeah I think that makes sense.